srcmedia/scss/search/_form.scss

```diff
@@ -12,4 +12,30 @@
         font-style: italic;
         font-size: 0.8rem;
     }
+
+    #members-form {
+        width: 100%;
+        text-align: center;
+    }
+
+    input[name=query] {
```

Comment:
I'm a big fan of this (the attribute selector) and have started doing it in similar situations also. No need for classes when you can be even more semantic using the attribute itself!